Description: Update test result
Subject: Update test result
 The upstream testsuite is using an embedded pygments version, which
 at the moment of writing this is 2.0pre. The version in Debian is
 slightly different (2.0rc1) and there are some minor mismatches. Most
 importantly, the Debian version is unable to find a good lexer for
 ambigous code "a". It is fixed by forcing it to use Ruby lexer.

 Already reported upstream https://github.com/tmm1/pygments.rb/issues/118
Author: Per Andersson <avtobiff@gmail.com>
---
--- a/test/test_pygments.rb
+++ b/test/test_pygments.rb
@@ -88,7 +88,7 @@
   end
 
   def test_highlight_works_with_single_character_input
-    code = P.highlight("a")
+    code = P.highlight("a", :lexer => 'ruby')
     assert_match 'a</span>', code
   end
 
@@ -283,5 +283,3 @@
     assert list['Html'][:aliases].include?('html')
   end
 end
-
-
